session3/styledComponents/src/Nav.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+import { StyledNav } from './Nav.style.js'
+
+class Nav extends Component {
```

Comment:
I don't see a use to this Component. It would be the same to just import `StyledNav` in `App.js`.

Reply:
Thanks @timgivois.
True, I was expecting it to grow bigger but also I moved the structure of the Header recently and it is in a kind of WIP.